cmd/genkey.go

```diff
@@ -17,7 +21,10 @@ import (
 	"github.com/foxboron/sbctl/fs"
 )
 
-const skipMicrosoftCerts = "skip-microsoft-certs-I-KNOW-WHAT-IM-DOING"
+const (
+	skipMicrosoftCertsFlag = "skip-microsoft-certs-I-KNOW-WHAT-IM-DOING"
```

Comment:
just a minor note: both flag needs documentation, but as I have https://github.com/kairos-io/kairos-docs/pull/168 flying, feel free to update that PR or comment there what's missing and I'll update it to cover these two

Reply:
I commented on the linked PR. Since enki has it's own repo, we could also document it in the README. Otoh, having docs in multiple locations can lead to outdated docs here and there.